package.json

```
@@ -3,13 +3,13 @@
   "version": "0.1.0",
   "private": true,
   "scripts": {
-    "dev": "next dev",
+    "dev": "NO_SW=true next dev",
```

```suggestion
    "dev": "next dev",
```

i'd rather this be removed and configurable by dev when running the command.

---

nvm. https://github.com/GoogleChrome/workbox/issues/1790